### PR TITLE
Fix shifted UI

### PR DESCRIPTION
**Pull request by mikeger**
- New AppRootViewController updated to host itself in the window correctly.

### CODE REVIEW

**Comment by mikeger** [comment]
Also initializes classy once (seems like it should work like that).

File: Wire-iOS/Sources/AppRootViewController.swift [hunk 1]
```diff
@@ -213,7 +220,7 @@ class AppRootViewController : UIViewController {
             overlayWindow.rootViewController = NotificationWindowRootViewController()
         }
         
-        if isClassyInitialized || appState == .authenticated(completedRegistration: false) || appState == .unauthenticated(error: nil) {
+        if !isClassyInitialized {
```
Comment (typfel) [inline Wire-iOS/Sources/AppRootViewController.swift]: The idea was not initialise classy when launching in the background but only when entering the authenticated or unauthenticated state.

**Comment by mikeger** [comment]
@typfel updated, there was another bug that app could have an inactive state, that was considered as app is running in the foreground.

**Comment by typfel** [comment]
@mikeger isn't the app inactive when the app is in foreground but some system message gets displayed?

**Comment by mikeger** [comment]
I was following this post: https://stackoverflow.com/questions/7937400/need-clarification-about-uiapplicationstate

Also when the app is starting due to push event it's inactive for me, so the classy was loaded, I had to change it to avoid this effect.

**Comment by typfel** [comment]
@mikeger ah ok 👍

**Pull request merged**